### PR TITLE
ARA custom report windows

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes:
- b/277376171

Changes proposed in this pull request:

- Create custom report windows from ARA handbook
- source: https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#bookmark=id.jqav7ioxrzta
- https://pr-6595-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/attribution-reporting/custom-report-windows/
- Have LGTM from Maud 6/28
- adding to toc in next PR

### CODE REVIEW

File: site/en/docs/privacy-sandbox/attribution-reporting/custom-report-windows/index.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+```json
+aggregatable_report_window
+```
+
+This will filter out conversions happening after a certain time frame.
+
+Aggregatable reports are sent within an hour of trigger; they're randomly delayed between 10 minutes and an hour.
+
+{% Aside %}
+In the table below, for all cases, if a conversion takes place **one day after the click/impression**, the browser will send the report **approximately an hour after the trigger**.
```
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/attribution-reporting/custom-report-windows/index.md:164]:
```suggestion
In the table below, for all cases, if a conversion takes place **one day after the click or impression**, the browser will send the report **approximately an hour after the trigger**.
```

Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/attribution-reporting/custom-report-windows/index.md:164]:
```suggestion
In the following table, for all cases, if a conversion takes place **one day after the click/impression**, the browser will send the report **approximately an hour after the trigger**.
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Some style guide nits (use 'following' rather than 'below' and spell out symbols.)

File: site/en/docs/privacy-sandbox/attribution-reporting/custom-report-windows/index.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Custom report windows'
+subhead: >
+  Custom report windows can provide faster event-level reports and selective reporting for aggregatable reports.
+description: >
+  Custom report windows can provide faster event-level reports and selective reporting for aggregatable reports.
+date: 2022-03-01
+updated: 2023-03-14
```
[the remark this answers is not in the document]
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/custom-report-windows/index.md:9]: It's not new. This is published content from google docs--https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit# and https://docs.google.com/document/d/1bU0a_njpDcRd9vDR0AJjwJjrf3Or8vAzyfuK8JZDEfo/view.
I've been asking about the dates. Rowan said to leave them

**Comment by nancymic2** [comment]
Feedback incorporated. thanks!

**Review by anusmitaray — APPROVED** [review]

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**